### PR TITLE
Add Design to welcome Email messages

### PR DESCRIPTION
**Pull request by BenjaminSsempala**
**_WHAT DOES THIS PR DO?_**
- Add Design to welcome Email messages

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [MOB-199](https://airqoteam.atlassian.net/browse/MOB-199)
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**
- Create an account on the mobile app using an email.
- You should get an email after 5 minutes

**_Screenshots

[MOB-199]: https://airqoteam.atlassian.net/browse/MOB-199?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @BenjaminSsempala

File: src/firebase/auth/functions/index.js [hunk 1]
```diff
@@ -144,7 +160,21 @@ async function checkIfUserExists(data) {
...
+        const userRef = firestoreDb.collection("airqo-app-users")
+            .doc(user.uid);
+        const userDoc = await userRef.get();
+
+        let firstName = userDoc.data().firstName;
+        if (firstName == null) {
+          firstName = "";
+        }
+        sendWelcomeEmail(email, firstName);
+      }, 300000);
```
Comment (Baalmart) [inline src/firebase/auth/functions/index.js:174]: @BenjaminSsempala , first take me through the purpose of the purpose of this setTimeout functionality....why did we need it? Thanks!

Comment (BenjaminSsempala) [reply src/firebase/auth/functions/index.js:174]: The set timeout gives the user time to finish setting up their account and input their account details, i.e. the name right after the signup process. We use the name of the user in the welcome email.

File: src/firebase/auth/functions/index.js [hunk 2]
```diff
@@ -144,7 +160,21 @@ async function checkIfUserExists(data) {
 exports.sendWelcomeEmail = functions.auth.user().onCreate((user) => {
   if (user.email !== null) {
     const email = user.email;
-    return sendWelcomeEmail(email);
+    try {
+      setTimeout(async () => {
+        const userRef = firestoreDb.collection("airqo-app-users")
```
[the remark this answers is not in the document]
Comment (BenjaminSsempala) [reply src/firebase/auth/functions/index.js]: @NoahNsimbe I've made the changes

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**